test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -143,8 +143,9 @@ public class Poco
 
             // BindingData is case insensitive. 
             // And queue name is normalized to lowercase. 
+            // Connection="" is same as Connection=null
             public const string QueueOutName = "qName-{XYZ}";
-            public void Func([QueueTrigger(QueueName)] Poco triggers,  [Queue(QueueOutName)] ICollector<string> q)
+            public void Func([QueueTrigger(QueueName, Connection ="")] Poco triggers,  [Queue(QueueOutName)] ICollector<string> q)
```

Comment:
Yes, the vast majority of are missing the Connection property altogether. So I just commandeered a case and tweaked it.  A little pair-wise testing.